### DISCUSSION TITLE
How can I set my sql string as start table?

### DISCUSSION THREAD

**Discussion by SebnemSeymen** [serenity-is/Serenity#6780] [Help]
There is a page I created in serenity, I want my list to be filled according to this query I wrote in Row.cs. But I couldn't set it. I'll be happy if you can help me.

TableName("[dbo].[DealerDetail]")]

This is my table name. I want the table in my sql query to come, not this table.

  [OuterApply("jManuelConfirmList", "SELECT C.MUSTERIID ,J.Value, (count(*) OVER(PARTITION BY C.MUSTERIID)) AS C On DealerDetail.MUSTERIID=C.MUSTERIID CROSS APPLY OPENJSON(C.Result) WITH (GroupDescription NVARCHAR(500) N'$.\"GroupDescription\"',Description NVARCHAR(500) N'$.\"Description\"' ,Value NVARCHAR(50) N'$.\"Value\"') AS J WHERE ISJSON(C.Result) > 0 AND DealerDetail.APPROVESTATUS=2 and J.Value = 6")]

**Comment by LSNicholls** [comment]
Can you create a view?

**Reply by SebnemSeymen** [reply]
Yes, i solved this problem.
